.github/workflows/update-docs-on-docs-commits.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Update generated parts of documentation on docs: commits'
```

Comment:
hey, not my repo, but please consider to not push this workflow here manually but rather push for having a global workflow, as we already have multiple repos that have the same workflow, so better to control it in `.github`

related issue https://github.com/asyncapi/.github/issues/242

Reply:
@derberg should I open a PR with this same file addressing https://github.com/asyncapi/.github/issues/242?

Reply:
I think yes. You should and also revert this file change for [update-docs-on-docs-commits.yml](https://github.com/asyncapi/glee/pull/548/files/fbe02953e6f405a889cb9f319119ed9c756e7305#diff-8eced5d3e71dce429a484047bea8fc9ed6289fc77807c8f9d3ed79df3541b2b1) 🌺

Reply:
> I think yes. You should and also revert this file change for [update-docs-on-docs-commits.yml](https://github.com/asyncapi/glee/pull/548/files/fbe02953e6f405a889cb9f319119ed9c756e7305#diff-8eced5d3e71dce429a484047bea8fc9ed6289fc77807c8f9d3ed79df3541b2b1) 🌺

@afzal442 I will revert this. But let's wait for the approval of this idea below from one of the maintainers... 
> open a PR with this same file addressing https://github.com/asyncapi/.github/issues/242?

Reply:
Hey @AnimeshKumar923 I had already talked to @derberg. He approved it. Plz go ahead

![image](https://github.com/asyncapi/glee/assets/11625672/07142b32-6cb1-4558-9871-e97778868e2a)

Reply:
@afzal442 Okay, got it. Will do the needful...

Reply:
I guess @Shurtu-gal opened a https://github.com/asyncapi/.github/pull/258 to solve this so I need not open a PR. I'm thinking of waiting for their PR approval.

cc: @derberg

Reply:
> You should and also revert this file
@AnimeshKumar923 could you also rmv this file? Else, LGTM!

Reply:
@afzal442 is it okay to remove? :eyes:

Reply:
> better to control it in .github

so what do you think about it? let me know.

Reply:
resolved with 494d5b2
cc: @afzal442